dist/gaia-header.js

```diff
@@ -548,27 +1143,43 @@ module.exports = Component.register('gaia-header', {
   }
 
   /**
-   * .flush-left
+   * .cant-center
    *
-   * When the fitted text is flush with the
-   * edge of the left edge of the container
-   * we pad it in a bit.
+   * In the rare event that there isn't
+   * enough room to center, it's best we
+   * fallback to left alignment, otherwise
+   * the title just looks broken.
    */
 
-  ::content h1.flush-left {
-    padding-left: 10px;
+  ::content h1.cant-center {
+    text-align: start;
```

Comment:
I think we agreed to keep `center` here

* best case (font size is correct): nearly no difference
* worst case (font size is incorrect): it doesn't look broken